pkg/node/node.go

```diff
@@ -1239,13 +1285,16 @@ func (s *Service) buildInitiatorsArray(initiators []string) []gopowerstore.Initi
 	var portType gopowerstore.InitiatorProtocolTypeEnum
 	if s.useFC {
 		portType = gopowerstore.InitiatorProtocolTypeEnumFC
-	} else {
+	} else if !s.useFC && s.useISCSI {
```

Comment:
!s.useFC check not needed here, as it is already in else if block

Reply:
Yes removed the !s.useFC condition from else if. Thanks